carbon-design-system/ibm-security
# chore: sort prop types for TagWall - WizardStep

## Pull request - 

### Affected issues

- Contributes to https://github.com/carbon-design-system/ibm-security/issues/973

### Proposed changes

- Sort the prop types

### Testing instructions

- run `yarn lint:js`
- verify there were no unintentional changes with the reordering as it as done by hand
